### PR TITLE
feat(chart): implement hover, refactor Chart

### PR DESCRIPTION
**Pull request by davidholyko**
#### Summary

- Work Type: Feature 
- Fixes: 

#### What I did

- Added on hover to chart squares
- Minor code refactor, use destructuring
-

#### Notes / Research / References

-
-

#### Before/After Preview

- Before:

- After: https://5f7c08961bae850008690a45--binge.netlify.app/#/chart/tt1486217

#### Checklist

- [ ] Branch is in format `TYPE/scope/description`
- [ ] PR title is in format `TYPE(scope): description`
- [ ] Commits are in semantic format
- [ ] Has Summary

### CODE REVIEW

File: roadmap.md [hunk 1]
```diff
@@ -1,16 +1,21 @@
+### Version 0.3.0 (in progress)
```
Comment (davidholyko) [inline roadmap.md:1]: Latest version should be at top

File: src/components/Chart/Chart.js [hunk 2]
```diff
@@ -2,21 +2,22 @@ import { _ } from '../../utils';
 import React, { useState } from 'react';
 
 import { ChartSquare } from './ChartSquare';
-import { generateSeasonsChart, generateYearsChart } from './ChartHelpers';
+import { generateSeasonsChart, generateYearsChart } from '../../utils';
```
Comment (davidholyko) [inline src/components/Chart/Chart.js:5]: moved file `ChartHelpers` to `utils/chart`

File: src/components/Chart/ChartSquare.js [hunk 3]
```diff
@@ -12,38 +19,36 @@ export const ChartSquare = ({ data, type }) => {
   const colorsGradient = gradient(COLOR_GRADIENT, 100);
 
   const style = {
-    gridColumnStart:
-      type === CHART_SQUARE_TYPES.YEAR_LABELS ? episodeNumber : null,
+    backgroundColor: type === EPISODE_DATA && getColor(rating, colorsGradient),
```
Comment (davidholyko) [inline src/components/Chart/ChartSquare.js:22]: alphabetize

File: src/components/Chart/ChartSquare.js [hunk 4]
```diff
@@ -12,38 +19,36 @@ export const ChartSquare = ({ data, type }) => {
...
-    'season-labels': type === CHART_SQUARE_TYPES.SEASON_LABELS,
-    'year-labels': type === CHART_SQUARE_TYPES.YEAR_LABELS,
+    'episode-data-square': type === EPISODE_DATA,
+    'episode-number-labels': type === EPISODE_NUMBER_LABELS,
+    'season-labels': type === SEASON_LABELS,
+    'year-labels': type === YEAR_LABELS,
   });
 
   const renderChartData = (inputType, inputData) => {
     let output = '';
```
Comment (davidholyko) [inline src/components/Chart/ChartSquare.js:38]: looking at this now, output refers to the text that is being displayed

File: src/api/api.js [hunk 5]
```diff
@@ -1,10 +1,17 @@
 import apiUrl from './apiConfig';
 import axios from 'axios';
 
-export const getEpisodes = async (searchTerm) => {
+export const getData = async (titleID) => {
   return await axios({
     method: 'GET',
-    url: apiUrl + `/episode-data/${searchTerm}`,
+    url: apiUrl + `/data/${titleID}`,
```
Comment (davidholyko) [inline src/api/api.js:7]: these routes in the backend should probably be renamed to `get-data`, `get-episode-data`, `get-meta-data` and so on

**Review by simonchen92 — APPROVED** [review]
LGTM

**Pull request merged**